tests/frontend/test_kibana_app/cypress/integration/features/filters/remove-filter-from-module-dashboard.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Validate that the added filter label is remove after click remove filter option
+   
+    As Wazuh user 
+    I want to set a new filter 
+    in order to manage them
+  
+   Scenario Outline: The user remove an added filter - Module - Dashboard
```

Comment:
minor: the user 'removed and added' a filter
or the user 'removes and adds' a filter

also, the part where we say '- module - dashboard' does this mean the localization of this functionality? can it be more clearly described?